index.html

```diff
@@ -1,67 +1,345 @@
-</head>
-<body>
-  <header>
-    <div class="status-bars">
-
-    </div>
-    <div class="app-bar">
-      <div class="toolbar">
-        <nav>
+  <head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Zema's Portfolio</title>
+    <link rel="stylesheet" href="styles.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;700&display=swap" rel="stylesheet">
+  </head>
+  <body>
+    <header>
+      <div class="status-bars">
+      </div>
+      <div class="app-bar">
+        <div class="toolbar">
           <div class="normal-button">
             <img class="burger" src="assets/burger.svg" alt="Menu icon.">
+            </div>
+            <div class="logo">
+              <span>My Logo</span>
+            </div>
+            <div class="toolbar-container">
+            </div>
           </div>
-
-        <a href="#home">
-          <div class="logo">
-            <span>My Logo</span>
-          </div>
-        </a>
-        <div class="toolbar-container">
-          
         </div>
-      </div>
-    </nav>
-
-    </div>
-  </header>
-  <main>
-    <section class="headline">
-      <div class="primary-text-2">
-        <h1>I'm Zermarden <br>
-        Glad to see <br>
-        you!</h1>
-      </div>      
-      <div class="social-media-icons">
-        <ul class="social-media-icons-list">
-          <li><a href="https://twitter.com/Zermarden"><img src="assets/twitter.svg" alt="Twitter" width="20" height="20"></a></li>
-          <li><a href="https://www.linkedin.com/in/zermarden/"><img src="assets/linkedin.svg" alt="Linkedin" width="20" height="20"></a></li>
-          <li><a href="https://medium.com/@zermarden"><img src="assets/m.svg" alt="Medium" width="20" height="20"></a></li>
-          <li><a href="https://github.com/Nedramrez"><img src="assets/git.svg" alt="GitHub" width="20" height="20"></a></li>
-          <li><a href="https://angel.co/u/zermarden-orazgaliyev"><img src="assets/finger.svg" alt="Peace" width="20" height="20"></a></li>
-      </ul>
-      </div>
-      <div class="social-media">
-        <div class="supporting-text">
-          <span>LET'S CONNECT!</span>
+        <section class="headline">
+          <h1 class="primary-text-2">I'm Zermarden <br>
+          Glad to see <br>
+          you!</h1>
+          <p class="primary-text">I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+          <div class="social-media">
+            <div class="supporting-text">
+              <span>LET'S CONNECT!</span>
+            </div>
```

Comment:
- [ ] Instead of a `<span>` please use a `<hx>` like a `<h4>` to describe the "LET'S CONNECT" call to action. This would be more semantically correct 😎